File: python/persistence/persondao.py
class PersonDao:
    def create(self, connection, person):
        stmt = connection.cursor()
        
        sql = "INSERT INTO person (firstName, lastName, email) values (%s, %s, %s)"
        
        data = (
            person.getFirstName(),
            person.getLastName(),
            person.getEmail()
        )

        stmt.execute(sql, data)

        connection.commit()

        stmt.close()
        connection.close()

    def read(self, connection):
        from entity.person import Person

        stmt = connection.cursor()

        sql = "SELECT * FROM person"

        stmt.execute(sql)
        results = stmt.fetchall()

        stmt.close()
        connection.close()
        
        list = []
        for result in results:
            person = Person()
            
            person.setId(result[0])
            person.setFirstName(result[1])
            person.setLastName(result[2])
            person.setEmail(result[3])
            
            list.append(person)
        
        return list

File: php/persistence/personDao.php
<?php
	class personDao{
		public function create($connection,$person){
			$sql = "insert into person(firstName,lastName,email) values ('".$person->getFirstName()."','".$person->getLastName()."','".$person->getEmail()."')";
			(mysqli_query($connection,$sql))?$query = true:$query = false;
			
			$connection->close();
			return $query;
		}

		public function read($connection){
			$sql = "select * from person";
			$result = mysqli_query($connection,$sql);
			$list = [];
			while($row = mysqli_fetch_assoc($result)){
				$person = new Person();

				$person->setId($row["id"]);
				$person->setFirstName($row["firstName"]);
				$person->setLastName($row["lastName"]);
				$person->setEmail($row["email"]);

				array_push($list,$person);
			}

			$connection->close();
			return $list;
		}

		public function update($connection,$person){
			$sql = "update person set firstName='".$person->getFirstName()."', lastName='".$person->getLastName()."' , email='".$person->getEmail()."' where id='".$person->getId()."'";
			echo $sql;

			(mysqli_query($connection,$sql))?$query = true:$query = false;
			
			$connection->close();
			return $query;
		}

		public function delete($connection,$person){
			$sql = "delete from person where id=".$person->getId();

			(mysqli_query($connection,$sql))?$query = true:$query = false;
			
			$connection->close();
			return $query;
		}
	}
?>
